samples/app-anonymous-users/csharp/ClientApp/src/router/router.js

```diff
@@ -10,7 +10,9 @@ import Configure from "../components/configure";
 import ShareToMeeting from "../components/share-to-meeting";
 import * as microsoftTeams from "@microsoft/teams-js";
 import ShareView from "../components/shareview";
-
+import FacebookAuthEnd from "../components/facebook-auth-end";
+import AuthStart from "../components/auth-start";
+import AuthEnd from "../components/auth-end";
```

Comment:
Shouldn't the `microsoftTeams.initialize();` below be `microsoftTeams.app.initialize();`? As this changed in v2 of the SDK

Reply:
Updated your feedback.